DonWorry/DonWorry/Views/CardCreationView/DecorateCard/DecorateCardView.swift

```diff
@@ -0,0 +1,251 @@
+//
+//  DecorateCardView.swift
+//  DonWorry
+//
+//  Created by 임영후 on 2022/06/03.
+//
+
+import SwiftUI
+import PhotosUI
+
+struct DecorateCardView: View {
+    
+//    @StateObject var vm = DecorateCardViewModel()
+    
+    @State private var decoCase: DecoCase = .account
+    
+    @State private var account: String = "1002-034-1234" // @@@@
+    @State private var color: CardColor = CardColor.blue// @@@@
+    @State private var images: [UIImage] = []// @@@@
+    @State private var showPhotoPicker = false
+    @State var isClicked: Bool = false
+    @State private var date = Date()
+    
+    //    let dismiss: () -> Void
+    
+    private let colorColumns = [GridItem](repeating: GridItem(spacing: 20), count: 5)
+    
+    var body: some View {
+        
+        NavigationView {
+            ZStack {
+                VStack(spacing: 20) {
+                    PreviewCardView(account: $account,
+                                    color: $color,
+                                    date: $date,
+                                    image: $images,
+                                    decoCase: $decoCase)
+                    .padding(.horizontal, 20)
+                    
+                    VStack(spacing: 20) {
+                        CustomPicker(selected: $decoCase)
+                            .padding(.horizontal, 20)
+                        Divider()
+                            .frame(height: 1)
+                            .padding(.horizontal, 40)
+                            .background(Color.grayEE)
+                        ZStack {
+                            if decoCase == .account {
+                                accountBox
+                            } else if decoCase == .color {
+                                colorBox
+                                    .padding(.horizontal, 30)
+                            } else if decoCase == .date {
+                                dateBox
+                                    .padding(.horizontal, 30)
+                            } else {
+                                imageBox
+                                    .padding(.horizontal, 40)
+                            }
```

Comment:
switch문으로 부탁드려요! enum 타입을 사용하는 이유는 가독성을 위해서라고 생각합니다!
```suggestion
                            switch decoCase {
                            case .color:
                                colorBox
                                    .padding(.horizontal, 30)
                            case .date:
                                dateBox
                                    .padding(.horizontal, 30)
                            case .account:
                                accountBox
                            case .image:
                                imageBox
                                    .padding(.horizontal, 40)
```

Reply:
반영